README.md

```diff
@@ -60,6 +60,27 @@ During the tutorial, don't hesitate to ask for help:
   can be posted.
 
 
+## Outline of the tutorial
+The tutorial will take you through stages to go a from a script (stage 0) to 
+the packaged application. Exercises will get you to build the most significant 
+portions of the differences between each stages but each of them is provided as 
+a separate folder are provided to make sure everyone to get to the end point:
+
+  - step 0: python script
+  - step 1: traits-ified script
```

Comment:
Could we call step 1 "a more robust script with traits"? Everyone may not know what "traits" is at this point, so including what it does may help.

Reply:
Good point, done.